man/systemd.netdev.xml

```diff
@@ -1268,6 +1268,27 @@
         </listitem>
       </varlistentry>
 
+      <varlistentry>
+        <term><varname>AdActorSysPrio=</varname></term>
+        <listitem>
+          <para>Specifies the 802.3ad system priority. Ranges [1-65535].</para>
```

Comment:
how come the word "actor" isn't mentioned in the description, but in the setting name?